frontend/src/component/Layout/Sidebar/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { memo, useMemo } from 'react';
+import { Box, Drawer, useMediaQuery } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+import { dispatch, useAppSelector } from '../../../store/hooks';
+import { viewsMiddleware, viewsSelector } from '../../../store/slices/menu';
+import MenuList from '../../MenuList';
+
+interface SidebarProps {
+  window?: Window;
+}
+
+const Sidebar = ({ window }: SidebarProps) => {
+  const theme = useTheme();
+  const matchUpMd = useMediaQuery(theme.breakpoints.up('md'));
+  const drawerWidth = 260;
+  const { drawerOpen } = useAppSelector(viewsSelector.menu);
+
+  const container = window !== undefined ? () => window.document.body : undefined;
```

Comment:
**question:** I kind of don't like how does it look, @RobMin WDYT?

Reply:
And I don't understand how can the window be undefined in a SPA.

Reply:
BTW even if you needed to write something like this
`window && () => window.document.body` is the same as `window !== undefined ? () => window.document.body : undefined`